src/main/java/step3/domain/Score.java

```diff
@@ -0,0 +1,19 @@
+package step3.domain;
+
+public class Score {
+    private final String symbol;
+    private String score;
+
+    public Score(String symbol) {
+        this.symbol = symbol;
+        this.score = symbol;
+    }
+
+    public void setScore() {
+        score += symbol;
+    }
```

Comment:
```suggestion
    public void increaseScore() {
        score += symbol;
    }
```

set 보다는 increase 와 같은 용어가, 메서드가 하고 있는 실제 역할을 더 잘 표현하고 있지 않나 제안해봅니다.

Reply:
동의합니다! 메서드명이나 필드명을 더 직관적으로 작성하는 습관을 들여야겠네요